packages/pipeline/src/concepts/queries/radar_orderbook_usd_prices.ts

```diff
@@ -0,0 +1,51 @@
+import { Connection } from 'typeorm';
+
+/**
+ * Async function that removes the concepts.radar_orderbook_usd_prices table if it exists
+ * @param connection connection to the db instance
+ */
+export async function deleteTableAsync(connection: Connection): Promise<void> {
+    await connection.query(`drop table if exists concepts.radar_orderbook_usd_prices`);
+}
+
+/**
+ * Async function that creates the concepts.radar_orderbook_usd_prices table
+ * @param connection connection to the db instance
+ */
+export async function createTableAsync(connection: Connection): Promise<void> {
+    await connection.query(
+        `
+create table concepts.radar_orderbook_usd_prices as
+with usd_prices as (
+    select * from raw.ohlcv_external
+    where to_symbol = 'USD' and end_time >= date_part('epoch', now() - interval '2 month')*1000
+), radar_orderbook as (
+    select * from raw.token_orderbook_snapshots
+    where source = 'radar' and observed_timestamp  > date_part('epoch', now() - interval '2 month')*1000
```

Comment:
first query is `>=` and this one is `>`, is that intentional?

Reply:
woops!